blog-detail-dark.php

```diff
@@ -12,300 +12,312 @@
     <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
```

Comment:
Your pull request says, you add 404 pages. But you edited blog-detail-dark.php as well. Why?

Reply:
I resolved a merge conflict bro. Please understand

Reply:
Okay. no vex. I am just trying to make sure things are not changed. U get ? thanks man for ur understanding

Reply:
No wahala bro. I get